openedx/features/content_tagging/models/base.py

```diff
@@ -81,6 +82,17 @@ def get_organizations(
             return list(Organization.objects.all())
         return [rel.org for rel in rels]
 
+    @classmethod
+    def get_admin_organizations(cls, taxonomy: Taxonomy, rel_type: RelType) -> List[Organization]:
+        """
+        Returns the list of Organizations which are admins of the taxonomy
+        """
+        rels = cls.objects.filter(
+            taxonomy=taxonomy,
+            rel_type=rel_type,
+        )
+        return [rel.org for rel in rels if rel.org is not None]
```

Comment:
You can make this more efficient by excluding the `null` orgs at the query level, and also select the `org` value in the join (instead of as a separate query per relationship):

```suggestion
        ).exclude(org=None).select_related('org')
        return [rel.org for rel in rels]
```

Reply:
Thank you! Done efd97f5d5457ae9e18012751064dfdaddfd53930

Reply:
Done 85df8e1b0dfdf477c13a6db8a4ea9f267c838a40